src/azure-firewall/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.0.0
+++++++
```

Comment:
Release notes should contain changes that impact customers, not internal changes.